src/sidecar/connectionStatusUtils.ts

```diff
@@ -61,6 +62,13 @@ export function connectionEventHandler(event: ConnectionEventBody) {
       break;
     }
     case ConnectionType.Ccloud:
+      // This function used to be called by a poller. Now we kick off whenever we get a ccloud connection
+      // event.
+      logger.info(
+        "connectionEventHandler: ccloud connection side effects firing, passing to watchCCloudConnectionStatus()",
+      );
+      reactToCCloudAuthState(connection);
```

Comment:
The core of the PR right here: whenever we get a websocket connection update for ccloud connection, call what the poller used to call.